hw6/series.py

```diff
@@ -0,0 +1,47 @@
+def fibonacci(n):
+    """Return the nth number of the fibonacci series"""
+    previous = 0
+    current = 1
+    next = 0
+    if n == 1:
+        return 0
+    elif n == 2:
+        return 1
+    if n > 2:
+        for number in range(n-2):
+            next = previous + current
+            previous = current
+            current = next
+    return current
```

Comment:
Most series in computer science start at 0. This implementation is a bit confusing having it start at 1. It also does not account for negative numbers or 0 being entered. This also applies to the other functions below.